beacon_node/execution_layer/src/test_utils/execution_block_generator.rs

```diff
@@ -349,9 +346,7 @@ impl<T: EthSpec> ExecutionBlockGenerator<T> {
     }
 
     pub fn insert_block(&mut self, block: Block<T>) -> Result<ExecutionBlockHash, String> {
-        if self.blocks.contains_key(&block.block_hash()) {
-            return Err(format!("{:?} is already known", block.block_hash()));
-        } else if block.parent_hash() != ExecutionBlockHash::zero()
+        if block.parent_hash() != ExecutionBlockHash::zero()
```

Comment:
Rejecting duplicates ends up being a big a pain in tests if we re-use a mock el between test beacon chains for example